test/helpers/application_helper_test.rb

```diff
@@ -45,4 +45,12 @@ class ApplicationHelperTest < ActionView::TestCase
       assert_equal expected_options, with_checked_options_at_top(options)
     end
   end
+
+  test "#govuk_tag returns a strong tag with the given text, and the class govuk-tag" do
+    assert_equal "<strong class=\"govuk-tag\">something</strong>", govuk_tag("something")
+  end
+
+  test "#govuk_tag returns a strong tag with the given text, and the classes govuk-tag plus any provided classes" do
+    assert_equal "<strong class=\"govuk-tag some-class another--class\">something</strong>", govuk_tag("something", "some-class another--class")
+  end
```

Comment:
Are you allowing the `classes` argument to be an array (given the `.compact`)? If so, is it worth adding a test for when an array is passed in?

Reply:
No, the intention was to allow a string of space-separated classes, or nil, and the compact is there to protect against the default nil value of `classes`.

Reply:
I wonder if this is making it too flexible, when we don't yet have any other uses for it.

I also notice that none of the other, newer, tables are using the tag class for their Yes/No strings. So we might want to remove it altogether...

Reply:
Ah, my Ruby is rusty: I'm confusing `.compact` and `.flat`!